docs/zope2book/AdvZPT.rst

```diff
@@ -1202,6 +1202,27 @@ caching for performance-critical applications.
 For more information on caching in the context of Zope, see the
 chapter entitled `Zope Services <ZopeServices.html>`_.
 
+Filesystem caching for Chameleon-based templates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Zope 4 introduced the `Chameleon HTML/XML template engine
+<https://chameleon.readthedocs.io/>`_ as new backend for Zope Page
+Templates. The Chameleon templating engine can compile templates and
+cache them on the file system for faster startup and execution.
+
+File system caching is activated by setting an environment variable
+named ``CHAMELEON_CACHE`` to the path of a folder on the filesystem
+where Chameleon can write its compiled template representation.
+
+Look for or add a section named ``environment`` in ``etc/wsgi.conf``
+and add a suitable filesystem path, for example::
+
+  <environment>
+    CHAMELEON_CACHE $INSTANCE/var

```

Comment:
Hm, this will pollute `/var` with many, maybe hundreds of files. I'd prefer to suggest to use a separate directory. I know `chameleon` requires this directory to exist beforehand, maybe this could be documented, too.

Reply:
@icemac I have changed the path to `$INSTANCE/var/cache` and added a note about creating that folder before starting Zope to the Zope Book chapter. Let me know what you think.

Reply:
I like it.